moment-timezone.d.ts

```diff
@@ -0,0 +1,68 @@
+// Type definitions for moment-timezone.js 0.2
+// Project: http://momentjs.com/timezone/
+// Definitions by: Michel Salib <https://github.com/michelsalib>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference path="../moment/moment.d.ts" />
+
+declare namespace moment {
+  interface MomentBuiltinFormat {
+    __momentBuiltinFormatBrand: any;
+  }
+
+  type MomentFormatSpecification = string | MomentBuiltinFormat | (string | MomentBuiltinFormat)[];

```

Comment:
Is the inconsistent indentation intentional?  (This and above vs. the interfaces below.)